### ISSUE TITLE
Accessibility: link purpose (in context) 2.4.4

### ISSUE THREAD

**Issue by chrismplus** [mplusmuseum/mplusmuseum-stories#174]
_Please cross-reference against Web Accessibility Evaluation Report, pp. 9-10._

**Issue 001: The language of the link text is inappropriate**
- [x] Links are written in English for Chinese pages

**Issue 002: Link text is not properly written**
- [x] Chris to follow-up with vendor
- [x] Rupert to investigate how emojis are being handled on screen readers

**Comment by chrismplus** [comment]
@rdunk 

Further information from HKBU: 

> Yes. There should be text appending the symbol. You may hide the additional text by CSS, please refer to this link,
>  
> C7: Using CSS to hide a portion of the link text:
> https://www.w3.org/TR/WCAG20-TECHS/C7.html

**Comment by rdunk** [comment]
I've added a feature in 2628a5ab9b1da5d1724e06a2f7d0de9c732bfe17 so that tag descriptors are now applied as a hidden label for screen readers if they are used.

For example the tag '🐔🐔🐔' with the EN descriptor 'chicken' and a blank TC descriptor will read 'chicken' if viewing in EN, but default to the tag title and read '🐔🐔🐔' in TC. i.e. there is no default language inheritance.

**Comment by chrismplus** [comment]
@rdunk — how does "🐔🐔🐔" get read in TC?

**Comment by rdunk** [comment]
I suppose that would depend on the screen reader software being used and how it interprets emojis. For example if I set mine to use a TC voice, it will read out Chinese characters correctly, but reads emoji descriptions out in English.

**Comment by chrismplus** [comment]
I see. So for this to work the English description should always be provided in Chinese...

**Comment by rdunk** [comment]
I'm not sure I follow. If alternative descriptions are set for both languages will read English descriptions if the site language is set to English and Chinese descriptions if set to Chinese.

If no alternative descriptions are set then it will just read out whatever the visual text is: so "🐔🐔🐔" would read however the screen reader software interprets those characters in whichever language the screen reader is set to.

**Comment by chrismplus** [comment]
Okay understood. This makes sense. Thanks.

**Comment by chrismplus** [comment]
This issue passed re-test. Moving to 'Done'. The rest of the solution lies in the authoring solution: 

![image](https://user-images.githubusercontent.com/26731347/35610081-ffeb917a-069a-11e8-8a91-d3e15024a3ff.png)
